packages/plugin-landing-page-graphql-playground/package.json

```diff
@@ -29,7 +29,7 @@
     "node": ">=14.0"
   },
   "dependencies": {
-    "@apollographql/graphql-playground-html": "1.6.29"
+    "@apollographql/graphql-playground-html": "^1.6.29"
```

Comment:
I sort of feel like for this one and the protobuf one (our forks) there's some argument to be made for having a strict link between the versions used (esp since both of these have really janky confusing build systems and I can imagine accidentally publishing a broken point release when trying to do a publish). I think it's OK either way, but there's at least some argument for keeping it the way it was before. Up to you.